commands/user.go

```diff
@@ -649,9 +649,13 @@ func deleteUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 			printer.PrintError("Unable to find user '" + args[i] + "'")
 			continue
 		}
-		if _, err := deleteUser(c, user); err != nil {
+		if res, err := deleteUser(c, user); err != nil {
 			printer.PrintError("Unable to delete user '" + user.Username + "' error: " + err.Error())
 		} else {
+			// res.StatusCode is checked for 202 to identify issues with file deletion.
+			if res.StatusCode == http.StatusAccepted {
+				printer.PrintT("There were issues with deleting profile image of the user. Please delete it manually. Id: '{{.Id}}'", user)
+			}
```

Comment:
I don't think we need `deleteUser`, so if you want to you could probably do...

```diff
diff --git a/commands/user.go b/commands/user.go
index 186060c0..33877f03 100644
--- a/commands/user.go
+++ b/commands/user.go
@@ -628,10 +628,6 @@ func resetUserMfaCmdF(c client.Client, cmd *cobra.Command, args []string) error
 	return nil
 }
 
-func deleteUser(c client.Client, user *model.User) (*model.Response, error) {
-	return c.PermanentDeleteUser(user.Id)
-}
-
 func deleteUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	confirmFlag, _ := cmd.Flags().GetBool("confirm")
 	if !confirmFlag {
@@ -649,7 +645,7 @@ func deleteUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 			printer.PrintError("Unable to find user '" + args[i] + "'")
 			continue
 		}
-		if res, err := deleteUser(c, user); err != nil {
+		if res, err := c.PermanentDeleteUser(user.Id); err != nil {
 			printer.PrintError("Unable to delete user '" + user.Username + "' error: " + err.Error())
 		} else {
 			// res.StatusCode is checked for 202 to identify issues with file deletion.
```

0/5 if you want to.